solutions/aspnet/config.json

```diff
@@ -6,7 +6,7 @@
     // The stack size of each ethread
     "StackMemSize": "400k",
     // The number of ethreads
-    "NumUserThreads": 16,
+    "NumUserThreads": 32,
```

Comment:
Can we delete the "NumUserThreads" setting altogether?

Reply:
I don't know. Maybe but I assume it runs faster with less theads.

Reply:
Oh I see. You're wondering if we can remove the setting altogether? If so, I think we should create an issue. It would be one less thing for users to worry about.

Reply:
Also we can do away with ``StackMemSize``

Reply:
I am wondering if this still works after deleting `NumUserThreads`. And if so, we can remove it from this PR. I don't want surprises when we run this test on a 8-core machine.

Reply:
I did not change this yet. I think we need to have a separate PR to consider removal of these settings since they don't just affect affinity.

Reply:
Affinity definitely makes it worse because dotnet runtime tends to create more threads when it sees more CPUs from getaffinity. That's why I ask for the change in the same PR.